src/Form/LocalistEntityForm.php

```diff
@@ -92,15 +96,68 @@ public function form(array $form, FormStateInterface $form_state) {
       '#required' => FALSE,
     ];
 
+
+    $form['extra_parameters_type'] = array(
+      '#type' => 'value',
+      '#value' => array('none' => t('None'),
+      'distinct' => t('Distinct'),
+      'all' => t('All Instances'))
+    );
+    $form['extra_parameters'] = array(
+      '#title' => t('Extra Localist URL Parameter'),
+      '#type' => 'select',
+      '#description' => '<ul><li>None: adds no parameters to localist query</li><li>Distinct: will return only next instance of an event</li><li>All Instances: returns all instances of an event</li>',
+      '#options' => $form['extra_parameters_type']['#value'],
+      '#default_value' => $localist_pull->extra_parameters,
+      '#required' => TRUE,
+    );
+    $form['date_label'] = array(
+      '#type' => 'label',
+      '#title' => $this->t('<br/>Date instructions:
+        <ul>
+          <li>Relative date  will be used if filled in</li>
+          <li>If relative date is blank this connection will use the static date</li>
+          <li>If there is no static date the default is today</li>
+        </ul>'),
+    );
+    $form['localist_relative_date'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Relative Date'),
+      '#default_value' => $localist_pull->localist_relative_date,
+      '#size' => 20,
+      '#maxlength' => 255,
+      '#description' => $this->t('CAUTION: be smart because we don\'t validate. Enter a relative date. This will be used as: date(\'Y-m-d\', strtotime("+30 days")); '),
```

Comment:
Nicer to user:
```suggestion
      '#description' => $this->t('Enter a relative date such as "+30 days". This will be used like: date(\'Y-m-d\', strtotime("+30 days")). CAUTION: The relative date must be a valid PHP relative date/time format.'),
```